include/net/xia_fib.h

```diff
@@ -194,6 +194,16 @@ struct xia_ppal_rt_iops {
 	 */
 	void (*fxid_init)(struct fib_xid *fxid, int table_id, int entry_type);
 
+	/* fxid_copy - copy @old_fxid, including FIB-specific data.
+	 *
+	 * NOTE
+	 *	This function must copy any FIB-specific data in addition
+	 *	to the fields from struct fib_xid. @old_fxid and @new_fxid
+	 *	should reference FIB entries previously allocated by
+	 *	fxid_ppal_alloc().
+	 */
+	void (*fxid_copy)(struct fib_xid *old_fxid, struct fib_xid *new_fxid);
```

Comment:
Would you mind swapping the position of the parameters? The assigning order is followed by the C library, and it bothers to go against it. A few examples: `memcpy()`, `memmove()`, and `strcpy()`.